packages/compiler/src/static-branching.ts

```diff
@@ -0,0 +1,21 @@
+export type AttributeValue =
```

Comment:
Since `AttributeValue` has already been defined in `compile.ts`, it seems redundant to me to redefine the same type.